tools/sqlcipher2hashcat.py

```diff
@@ -0,0 +1,7 @@
+from base64 import b64encode
+import sys
+
+database = open(sys.argv[1], "rb").read(272)
```

Comment:
here is the `open()` function call without any further sanity checks.

Furthermore, all our `*.py`, `*.pl`, `*.sh` ... files are executable (the file flag chmod +x ), we should also make this file executable